api/models.py

```diff
@@ -62,3 +62,17 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateField(blank=False, null=False)
```

Comment:
I dont get it. If I delete start_date, start_time, end_date, end_time ... then there wont be any info on website when the event starts. I agree it can bi both on one field (models.DateTimeField), but that's what I did at first time and it was wrong ..